inngest-core/src/main/kotlin/com/inngest/Step.kt

```diff
@@ -77,4 +84,43 @@ class Step(val state: State) {
             throw StepInterruptSleepException(id, hashedId, "${durationInSeconds}s")
         }
     }
+
+    /**
+     * Sends multiple event to Inngest
+     *
+     * @param id Unique step id for memoization.
+     * @param event An event payload object.
+     *
+     */
+
+    fun sendEvent(
+        id: String,
+        event: InngestEvent,
+    ) = sendEvent(id, arrayOf(event))
+
+    /**
+     * Sends a single event to Inngest
+     *
+     * @param id Unique step id for memoization.
+     * @param events An array of event payload objects.
+     *
+     */
+    fun sendEvent(
+        id: String,
+        events: Array<InngestEvent>,
+    ) {
```

Comment:
the return type should be similar to `send` itself.
https://www.inngest.com/docs/reference/functions/step-send-event#step-send-event-id-event-payload-event-payload-promise-ids-string

Reply:
gotcha, I'll address them in this PR 👍

Reply:
Turns out this is a bit tricky, we left a TODO to investigate later